test/unit/services/test_catalogue_category.py

```diff
@@ -698,6 +698,9 @@ def test_update_change_catalogue_item_properties_when_has_children(
     )
     # pylint: enable=duplicate-code
 
+    # Mock so child elements found
+    catalogue_category_repository_mock.has_child_elements.return_value = True
```

Comment:
Why is this needed?

Reply:
This was removed in #123, see comment here https://github.com/ral-facilities/inventory-management-system-api/pull/133#discussion_r1439369322.

Reply:
Oh I see, I suggest that `develop` gets merged into this branch then. Ping @MatteoGuarnaccia5.

Reply:
@VKTB Ah sorry no I meant it was removed by accident, it's not in develop as that PR was merged after Joshua approved that PR, but before I had approved it, so I asked it to be re-added here as I had requested in that PR.

Reply:
@joelvdavies Ah sorry, I now get what you meant.